qupulse/program/linspace.py

```diff
@@ -194,9 +194,26 @@ def with_repetition(self, repetition_count: RepetitionCount,
     def with_sequence(self,
                       measurements: Optional[Sequence[MeasurementWindow]] = None) -> ContextManager['ProgramBuilder']:
         yield self
-
+        
+    @contextlib.contextmanager
     def new_subprogram(self, global_transformation: 'Transformation' = None) -> ContextManager['ProgramBuilder']:
-        raise NotImplementedError('Not implemented yet (postponed)')
+        
+        inner_builder = LinSpaceBuilder(self._idx_to_name)
+        yield inner_builder
+        inner_program = inner_builder.to_program()
+
+        if inner_program is not None:
+            #measurements not yet included
```

Comment:
The linspace builder ignores measurements. The idea is to have a dedicated `MeasurementBuilder` to extract them.

Reply:
ok